src/restClient.js

```diff
@@ -0,0 +1,719 @@
+/**
+ * (c) Copyright [yyyy] EntIT Software LLC
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*  http://www.apache.org/licenses/LICENSE-2.0
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+
+/**
+ * REST wrapper around the swagger generated code for SSC.
+ */
+"use strict";
+const dotenv = require('dotenv').config();//must be before config
+import configLoader from '../config';
+import Swagger from 'swagger-client';
+import async from 'async';
+import request from 'request'
+import fs from "fs";
+import http from 'http';
+
+const config = configLoader.loadEnv();
+
```

Comment:
Does this file contain some utility / common functions used by other modules? Or it also contains examples of API usage? if latter, it might be better to split each example into individual file. In this case it will be easier for people to search appropriate example by file name.